src/extension.ts

```diff
@@ -1,11 +1,21 @@
-import * as dnode from 'dnode';
+import * as dnode from 'dnode'

```

Comment:
Actually, I would prefer to have a semicolon at the end of each line here. I'm not sure why it was missing from so many of the import lines.